include/boost/phoenix/stl/tuple.hpp

```diff
@@ -0,0 +1,100 @@
+/*==============================================================================
+    Copyright (c) 2005-2008 Hartmut Kaiser
+    Copyright (c) 2005-2010 Joel de Guzman
+    Copyright (c) 2010 Thomas Heller
+    Copyright (c) 2021 Beojan Stanislaus
+
+    Distributed under the Boost Software License, Version 1.0. (See accompanying
+    file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+==============================================================================*/
+
+#ifndef BOOST_PHOENIX_STL_TUPLE_H_
+#define BOOST_PHOENIX_STL_TUPLE_H_
+#if __cplusplus >= 201402L || (defined(_MSC_FULL_VER) && _MSC_FULL_VER >= 190024210)
+
+#include <tuple>
+
+#include <boost/phoenix/core/argument.hpp>
+#include <boost/phoenix/core/call.hpp>
+#include <boost/phoenix/core/expression.hpp>
+#include <boost/phoenix/core/limits.hpp>
+#include <boost/phoenix/core/meta_grammar.hpp>
+
+// Lazy functions for std::tuple (and similar)
+// get will work wherever it is accessible through ADL
+// Cribbing from functions in object
+
+namespace boost {
```

Comment:
Please copy the coding style of the codebase. See the other header files.

Reply:
I think I've addressed this. Let me know if there are other changes required.